adobe/json.hpp

```diff
@@ -200,11 +200,10 @@ class json_parser {
         frac();
         exp();
 
-        const auto p_len = p_ - p;
         char* p_end = nullptr;
         double value = std::strtod(p, &p_end);
         require(std::isfinite(value), "finite number");
-        ADOBE_ASSERT(p_len == (p_end - p) && "std::strtod() failure");
+        ADOBE_ASSERT((p_ - p) == (p_end - p) && "std::strtod() failure");
```

Comment:
Surely not needed for C++20?  Is this an improvement?  It's not great code either way, but at least the old code let you know that `p_ - p` is somehow the length of `p` (whatever that is).

Reply:
The unused `p_len` was generating warnings in release builds.